src/admin-views/help-ticketing.php

```diff
@@ -5,9 +5,12 @@
 $faqs               = tribe( Tribe__Admin__Help_Page::class )->get_ticketing_faqs();
 $extensions         = tribe( Tribe__Admin__Help_Page::class )->get_ticketing_extensions();
 $ticketing_products = tribe( Tribe__Admin__Help_Page::class )->get_ticketing_products();
+$is_tickets_help_page = tribe( Tribe__Admin__Help_Page::class )->is_ticketing_current_help_page();
+
+$tribe_ticketing_style = $is_tickets_help_page ? 'style="display:block;"' : 'style="display:none;"';
```

Comment:
The same comments as above regarding prefix and the value that we store on the variable without the `style="...`

Reply:
Thanks for noticing this. This is fixed now